lib/commands/actions.js

```diff
@@ -166,6 +167,16 @@ commands.getScreenshot = async function () {
     await fs.unlink(localFile);
   }
   await this.adb.pull(png, localFile);
+  let screenOrientation = await this.adb.getScreenOrientation();
+  if (screenOrientation) {
+    jimp.read(localFile).then(function (image) {
```

Comment:
sure, no problem! This is how you'd do it (assuming `import B from 'bluebird'` is at the top somewhere):

```js
let screenOrientation = await this.adb.getScreenOrientation();
let image = await jimp.read(localFile);
try {
  image = await image.rotate(90 * screenOrientation);
} catch (err) {
  log.warn(`Could not rotate screenshot due to error: ${err}`);
}
let b64data = (await B.promisify(image.getBuffer)(jimp.MIME_PNG))
                .toString('base64');
```